### DISCUSSION TITLE
Welcome to xmlsec Discussions!

### DISCUSSION THREAD

**Discussion by lsh123** [lsh123/xmlsec#409] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jefmason** [comment]
I work at Siemens with a number of 'legacy' C/C++ applications that use openssl/xmlsec.  I am currently involved in evaluating a port to openssl3.  My involvement is to use existing functionality in these packages moving into the future

**Comment by jefmason** [comment]
I just noticed that xmlsec does not release the xmlsec/openssl include files when only a static library is built on windows.  I modified the win32 Makefile.msvc to add targets to create this directory and populate it when  $(BINDIR)\$(XMLSEC_OPENSSL_A)  exists.  I now get these files released in my build and can use them.  I did not modify any other targets like uninstall, only the install target.  I believe this is the correct solution to an oversight.

**Reply by lsh123** [reply]
could you please create an issue on github and if possible share your changes as a PR?

**Comment by jefmason** [comment]
I am new to filing things in github so will attempt to do this.  Thanks.

**Reply by postboy** [reply]
For the history purposes: this discussion continued in https://github.com/lsh123/xmlsec/issues/420.

**Comment by bernardohoehl-web** [comment]
Hello! I have a Mac Os X app that uses XMLSEC library to sign an xml string using a p12 certificate. It is old code and still uses xmlsec1-1.2.20. I am having 32 Xcode build errors related to xmlsec library, and so I would like to update my XMLSEC Library used to newest Apple M* (ARM64 or AArch64) chips. I am willing to hire a more experienced programmer willing to help me. Can you help me compile XMLSEC Library to the newest Apple's chips?

**Reply by lsh123** [reply]
OSX builds are part of XMLSec commit checks so it should “just work”:

https://github.com/lsh123/xmlsec/actions/runs/19788968992/job/56699348014

Build steps:

https://github.com/lsh123/xmlsec/blob/master/.github/workflows/make-check.yml#L108

**Comment by lsh123** [comment]
Actually, just use homebrew:

1/ install brew: https://brew.sh/

2/ install dependencies and xmlsec:

brew upgrade

brew install libxml2 libxslt
brew install openssl 
brew install libxmlsec1
